### PR TITLE
[WIP] Create a test-suite to identify tests that fail to execute

### PR DESCRIPTION
**Pull request by iamaspacecow**
Next step is to triage the tests into tests that should execute and tests that shouldn't, and not count the latter as failures
Not complete, but first step is done

### CODE REVIEW

**Comment by gvanrossum** [comment]
You know you can run the tests on your own machine too, right? There are hints in README.md.

File: mypy/waiter.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def status(self) -> Optional[int]:
     def read_output(self) -> str:
         with open(self.outfile.name, 'rb') as file:
             # Assume it's ascii to avoid unicode headaches (and portability issues).
-            return file.read().decode('ascii')
+            return file.read().decode('utf-8')
```
Comment (gvanrossum) [inline mypy/waiter.py]: Hm, the ASCII was probably intentional (so we won't have mystery Unicode characters in tests). What prompted this change?

Comment (iamaspacecow) [inline mypy/waiter.py]: Probably I had mystery Unicode characters in my test somewhere. If I run the test-suite (with my test added), it raised and Exception about being unable to convert to unicode. I should go over my tests and fix the root error instead there

**Comment by iamaspacecow** [comment]
The failing unit test here is related to mypy failing on my new runner code.
I submitted PR's to typeshed to get the necessary standard library definitions that would enable my code to 
pass.

**Comment by iamaspacecow** [comment]
I should also not that this PR is related to fixing issue #687

**Comment by gvanrossum** [comment]
I am closing this for lack of progress. Max, if you want to work on this again, please re-open the PR!

**Pull request closed without merging**